vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -493,19 +496,6 @@ func (c completedConfig) New(name string, delegationTarget DelegationTarget) (*G
 		maxRequestBodyBytes:          c.MaxRequestBodyBytes,
 	}
 
-	for {
-		if c.JSONPatchMaxCopyBytes <= 0 {
-			break
-		}
-		existing := atomic.LoadInt64(&jsonpatch.AccumulatedCopySizeLimit)
-		if existing > 0 && existing < c.JSONPatchMaxCopyBytes {
-			break
-		}
-		if atomic.CompareAndSwapInt64(&jsonpatch.AccumulatedCopySizeLimit, existing, c.JSONPatchMaxCopyBytes) {
-			break
-		}
-	}
-
```

Comment:
this shouldn't be here. It was added in the first UPSTREAM commit and is removed here again.

Reply:
Add this back in?

for {
		if c.JSONPatchMaxCopyBytes <= 0 {
			break
		}
		existing := atomic.LoadInt64(&jsonpatch.AccumulatedCopySizeLimit)
		if existing > 0 && existing < c.JSONPatchMaxCopyBytes {
			break
		}
		if atomic.CompareAndSwapInt64(&jsonpatch.AccumulatedCopySizeLimit, existing, c.JSONPatchMaxCopyBytes) {
			break
		}
}

Reply:
yes

Reply:
This needs a later version of json-patch as current version does not include AccumulatedCopySizeLimit. What is the correct way to do that? Use the sha from the 3.10 glide.lock?

Reply:
yes, maybe the 3.10 version.